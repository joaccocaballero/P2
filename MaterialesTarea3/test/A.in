buscarSubarbol 20
linealizacion
esVacioAbb
imprimirAbb
copiaAbb
consAbb (20, 20.0) 10 30
esPerfecto
menores 20
insertarPalabra palas
imprimirPalabrasCortas 2
Fin