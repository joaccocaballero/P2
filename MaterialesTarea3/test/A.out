1>20 no está en el abb.
2>
3>El abb es vacío.
4>
5>
6>
(20,20.00)
7>El abb es perfecto.
8>
9>Insertando palas.
10>
11>Fin.
